tools/accuracy_checker/configs/im2latex-0001.yml

```diff
@@ -0,0 +1,65 @@
+evaluations:
+  - name: im2latex-medium-0002
+    module: custom_evaluators.im2latex_evaluator.Im2latexEvaluator
+    module_config:
+      network_info:  
+        recognizer_encoder:
+          model:   intel/im2latex-medium-scanned-0002/encoder.xml
+          weights: intel/im2latex-medium-scanned-0002/encoder.bin
+
+        recognizer_decoder:
+          model:   intel/im2latex-medium-scanned-0002/decode_step.xml
+          weights: intel/im2latex-medium-scanned-0002/decode_step.bin
+        max_seq_len: '192'
+
+      launchers:
+        - framework: dlsdk
+          tags:
+            - FP32
+
+      datasets:
+        - name: im2latex_medium_photographed
+
+          preprocessing:
+            - type: resize
+              aspect_ratio_scale: scale_factor
+              scale: 0.625
+            - type: crop
+              dst_width: 1400
+              dst_height: 160
+              no_resize: True
+            - type: padding
+              pad_type: right_bottom
+              dst_width: 1400
+              dst_height: 160
+              pad_value: '(255, 255, 255)'
+            - type: normalization
+              std: 255
+              mean: 0
+            
+          metrics:
+            - type: im2latex_match_images_metric
+              max_pixel_column_diff: 0
+
+        - name: im2latex_medium_rendered
+
+          preprocessing:
+            - type: crop
+              dst_width: 1400
+              dst_height: 160
+              no_resize: True
+            - type: padding
+              pad_type: right_bottom
+              dst_width: 1400
+              dst_height: 160
+              pad_value: '(255, 255, 255)'
+            - type: normalization
+              std: 255
+              mean: 0
```

Comment:
usually, when model published in OMZ as IR, normalization is done during model conversion via MO. if it is your case, please remove.